sdks/python/apache_beam/examples/wordcount.py

```diff
@@ -65,8 +74,12 @@ def process(self, element):
       self.word_lengths_dist.update(len(w))
     return words
 
+  def to_runner_api_parameter(self, unused_context):
+    pass
+
 
 def run(argv=None):
+  # type: (Optional[Sequence[str]]) -> None
```

Comment:
One of the benefits of gradual typing is that types are optional. It's unclear that this hint (among others) provides a net positive value to (to merit its verbosity).

Reply:
(And I realize this is a WIP, but we should be very careful about changing the wordcount (and other examples) to keep then concise as an introduction to the Beam APIs.

Reply:
Sorry, I should have clarified that the changes to wordcount really amount to tests, primarily of the mypy plugin.  Ultimately, we'll want a separate example to demonstrate typing.  I'll get this reverted soon.

Reply:
Sounds good. Thanks for the clarification.